src/mcp/tools/index.ts

```diff
@@ -26,19 +26,40 @@ export function availableTools(fixedRoot: boolean, activeFeatures?: ServerFeatur
 }
 
 const tools: Record<ServerFeature, ServerTool[]> = {
-  firestore: addPrefixToToolName("firestore_", firestoreTools),
-  auth: addPrefixToToolName("auth_", authTools),
-  dataconnect: addPrefixToToolName("dataconnect_", dataconnectTools),
-  storage: addPrefixToToolName("storage_", storageTools),
-  messaging: addPrefixToToolName("messaging_", messagingTools),
+  firestore: addProductGroupPrefix("firestore", firestoreTools),
+  auth: addProductGroupPrefix("auth", authTools),
+  dataconnect: addProductGroupPrefix("dataconnect", dataconnectTools),
+  storage: addProductGroupPrefix("storage", storageTools),
+  messaging: addProductGroupPrefix("messaging", messagingTools),
 };
 
-function addPrefixToToolName(prefix: string, tools: ServerTool[]): ServerTool[] {
+function addProductGroupPrefix(productGroup: string, tools: ServerTool[]): ServerTool[] {
   return tools.map((tool) => ({
     ...tool,
     mcp: {
       ...tool.mcp,
-      name: `${prefix}${tool.mcp.name}`,
+      name: `${productGroup}_${tool.mcp.name}`,
+      productGroup,
     },
   }));
 }
+
+/**
+ * Generates a markdown table of all available tools and their descriptions.
+ * This is used for generating documentation.
+ */
+export function markdownDocsOfTools(): string {
+  const allTools = availableTools(false, []);
+  let doc = `
+| Tool Name                        | Feature Group | Description         |
+| -------------------------------- | ------------- | ------------------- |`;
```

Comment:
If you're not actually going to consistently pad it in the content, may as well just drop the extra spaces.